lib/contact_screen.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:contacts_service/contacts_service.dart';
+import 'package:flutter/material.dart';
+import 'package:kilvish/common_widgets.dart';
+import 'package:kilvish/style.dart';
+import 'package:permission_handler/permission_handler.dart';
+
+
+
+class ContactScreen extends StatefulWidget {
+  const ContactScreen({Key? key}) : super(key: key);
+
+  @override
+  State<ContactScreen> createState() => _ContactScreenState();
+}
+
+class _ContactScreenState extends State<ContactScreen> {
+
+  List<Contact> contacts = [];
```

Comment:
This is incomplete code .. some contacts will already have account on kilvish & we can hardcode that information here in the code. 

while displaying those contacts, we need to show their kilvish id on the UI